service/src/main/java/com/derpgroup/derpwizard/manager/AbstractManager.java

```diff
@@ -64,6 +82,31 @@ public void handleRequest(@NonNull VoiceInput voiceInput, @NonNull SsmlDocumentB
     }
   }
 
+  private void appendToConversationHistory(@NonNull VoiceInput voiceInput) {
+    CommonMetadata metadata = voiceInput.getMetadata();
+    List<ConversationHistoryEntry> conversationHistory = metadata.getConversationHistory();
+    if(conversationHistory == null){
+      conversationHistory = new ArrayList<ConversationHistoryEntry>();
+    }
+    CommonMetadata metadataClone = null;
+    try {
+      ObjectMapper mapper = new ObjectMapper();
+      mapper.registerModule(mapperModule);
+      String metadataString = mapper.writeValueAsString(metadata);
+      metadataClone = mapper.readValue(metadataString, CommonMetadata.class);
```

Comment:
Could these two lines be changed to this?
metadataClone = mapper.convertValue(metadata,  CommonMetadata.class);

I don't know enough about Jackson to know if it behaves fundamentally differently. It doesn't seem to require a try/catch either for some reason. Which seems odd.